pyproject.toml

```
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+
+
+[tool.setuptools]
+packages = ["usd_qtpy"]
+
+
+[project]
+name = "usd-qtpy"
+authors = [
+    {name = "BigRoy"},
+    {name = "Hannes"},
+]
+description = "Python Qt components for building custom USD tools."
+readme = "README.md"
+requires-python = ">=3.4"
+keywords = ["usd", "qt"]
+license = { file = "LICENSE" }
+classifiers = [
+    "Programming Language :: Python :: 3.7",
+]
```

Should we set the classifiers like this?
```suggestion
classifiers = [
    "Programming Language :: Python :: 3.9",
    "License :: OSI Approved :: MIT License",
    "Operating System :: OS Independent",
]
```

---

I'm not sure, if the pip install still works why not :)